packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -144,6 +168,7 @@ export default class OperatorModeStackItem extends Component<Signature> {
           style={{cssVar 
             boxel-header-icon-width='30px' 
             boxel-header-icon-height='30px'
+            boxel-header-text-color=(if this.isHoverOnRealmIcon 'var(--boxel-cyan)' 'var(--boxel-dark)')
```

Comment:
Does the design call for cyan? It doesn’t immediately convey “this is the realm name” to me, is there some way to make that clear?

Reply:
![Screenshot 2023-07-12 at 11 54 02](https://github.com/cardstack/boxel/assets/12637010/49f7acf2-a07e-43bf-96b3-daaf6ce25d75)

The text color looks the same as the other icons with cyan color. We can change the wording later to make it clear.